src/webapi/node.rs

```diff
@@ -858,4 +872,28 @@ mod tests {
         assert!(format!("{}", err).contains("requires a single root node"));
         assert!(Node::from_html("<di").is_err());
     }
+
+    #[test]
+    fn from_html_trimmed() {
+        let node = Node::from_html("
+          <div>Some text, horray!</div>
```

Comment:
Please make the newlines explicit in these tests. (Replace actual newlines with `\n`.)

Reply:
Done.